project-docs/stumptown-writers-guide.md

```diff
@@ -0,0 +1,697 @@
+* [link text 2](link target 2): short description for the link 2
+...
+
+## Title of another list of Links
+
+* [link text 3](link target 3): short description for the link 3
+* [link text 4](link target 4): short description for the link 4
+...
+```
+
+**Note: we don't currently support short descriptions for guide pages. But we should.**
+
+Each "link list" object may be specified in one of three ways:
+
+-   as a directory. An object that consists of two properties: `title`, which is a string that will be used as the list title, and `directory`, which is a path to a directory. Stumptown will look for all pages immediately inside the given directory, and make links to them. For example, if you give it "/content/html/reference/elements" it will make a list of all the HTML elements.
+
+-   as an explicit list of pages. An object that consists of two properties: `title`, which is a string that will be used as the list title, and `pages`, which is an array of paths to pages. Stumptown will make links to all the pages listed.
+
+-   as a `chapter_list`. An object that consists of one property: `chapter_list`, which is a path to a "Chapter list" YAML file. Stumptown will make links to all the pages listed in the chapter_list. This is really just a reusable version of the explicit list of pages.
+
+For example:
+
+```
+link_lists:
+  - title: Index of HTML elements
+    directory: "/content/html/reference/elements"
+  - title: Some guide pages
+    pages:
+      - "/content/html/guides/Block-level_elements"
+      - "/content/html/guides/Inline_elements"
+  - chapter_list: "/content/learn/html/Introduction_to_HTML.yaml"
+```
+
+This specifies three link lists.
+
+-   The first has the title "Index of HTML elements" and contains links to all the pages under "/content/html/reference/elements".
+-   The second has the title "Some guide pages", and contains links to the two pages listed.
+-   the third as the title and link list that's given in the referenced chapter list.
+
+## Landing pages
+
+From MDN's own meta-docs:
+
+> A landing page serves as a menu, of sorts, for its subpages, and is therefore primarily a navigation page. A landing page layout is typically used for the root page of a tree of pages about a particular topic. It opens with a brief summary of the topic, then presents a structured list of links to its subpages, and optionally, additional material that be useful to the reader... The list of subpages can be generated automatically...
+
+So typically they are made of some prose content on some high-level topic like HTML, followed by a collection of links to the pages comprising the docs for this topic.
+
+Stumptown supports landing pages using Markdown files with YAML front matter. Although their front matter does list a `recipe: landing-page` property, the recipe itself isn't used in building the page.
+
+**Note: we should just make landing pages recipe-driven like the reference pages.**
```

Comment:
I think this would be useful so that landing pages appear consistent throughout MDN.

Reply:
Definitely; will also make it easier to build the site when these are recipe driven.

Reply:
Filed https://github.com/mdn/stumptown-content/pull/206 for this.